ArduCopter/Parameters.cpp

```diff
@@ -1089,6 +1089,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_GROUPINFO("TERRAIN_MARGIN", 47, ParametersG2, terrain_margin, 10.0),
 #endif
 
+    // @Param: AUTO_RTL
```

Comment:
I think we can skip this parameter can't we?  That would be nice 'cuz I'm not really a fan of the name.  It's not terrible but it is all on it's own and is the only parameter starting with "AUTO_".   Not a blocker but if possible I'd like to do without this parameter for now.

Reply:
Yeah, the param is pointless in this PR, I have 3 other methods to add so will need it, but that will be a second PR.